h/static/scripts/account/account-controller.coffee

```diff
@@ -1,8 +1,8 @@
 class AccountController
   @inject = [  '$scope', '$filter',
-               'auth', 'flash', 'formHelpers', 'identity', 'session']
+               'auth', 'formHelpers', 'identity', 'flash', 'session']
```

Comment:
A tiny niggle, but let's avoid no-op changes like this in the history, if possible.